supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJob.scala

```diff
@@ -120,8 +126,9 @@ final case class AwsBatchJob(jobDescriptor: BackendJobDescriptor, // WDL/CWL
     val replaced = commandScript.replaceAllLiterally(AwsBatchWorkingDisk.MountPoint.pathAsString, workDir)
     val insertionPoint = replaced.indexOf("\n", replaced.indexOf("#!")) +1 //just after the new line after the shebang!
 
-    //generate a series of s3 copy statements to copy any s3 files into the container
-    val inputCopyCommand = inputs.map {
+    /* generate a series of s3 copy statements to copy any s3 files into the container. We randomize the order
+       so that large scatters don't all attempt to copy the same thing at the same time. */
```

Comment:
TOL A worthy goal but I'm not sure how much randomizing would generally be a help with that